Sources/Persistence/Metadata.swift

```diff
@@ -4,10 +4,8 @@ public struct Metadata: Equatable, Codable {
     let sdkVersion: String?
     let iosVersion: String?
     let macosVersion: String?
-
-    private static let userDefaults = UserDefaults(suiteName: Constants.UserDefaults.suiteName)
-
-    static func getCurrentMetadata() -> Metadata {
+    
+    static func getCurrent() -> Metadata {
```

Comment:
The following way of providing a property for singleton classes is widely used in Swift:

```
static var current: Metadata = {
    // Instantiate here.
}()
```

Reply:
👍 thank you!